index.html

```diff
@@ -8,6 +8,37 @@
     <title>Personal Portfolio</title>
 </head>
 <body>
-    
+    <nav class="toolbar">
+        <figure class="logo-wrapper">
+            <img src="./images/My-Logo.png" alt="">
+        </figure>
+
+        <figure class="menu-icon">
+            <img src="./images/Union.png" alt="">
+        </figure>
+    </nav>
```

Comment:
**REQUIRED:** The container of the toolbar section must be `header`, not `nav`, because as you can see toolbar is not parent of any link.